tests/unit/amo/components/TestGetFirefoxButton.js

```diff
@@ -135,33 +134,48 @@ describe(__filename, () => {
         });
       });
 
-      it.each([true, false])(
```

Comment:
Why did you remove the `it.each` and replace it with two separate test cases that do the same thing? Can't we just use `it.each` here and add a comment explaining why there is no variation in the expectations?

Reply:
ah yes, I removed one of the cases and then re-added it later..